create_phone_number.py

```diff
@@ -0,0 +1,3 @@
+def create_phone_number(n):
+    result_str = ''.join([str(digit) for digit in n])
+    return f'({result_str[:3]}) {result_str[3:6]}-{result_str[6:]}'
```

Comment:
In general don't use single character parameter/variable names.  n --> number or num.  Even if this is the way they have it in the code challenge, a plain parameter can be renamed anything you like.  kwargs would be a different story.

Variables should be named after what kind of information they hold rather than the python type they are.  result_str --> result

Great use of join and the list comprehension.

Reply:
> Variables should be named after what kind of information they hold rather than the python type they are. result_str --> result